README.md

```diff
@@ -1319,23 +1321,63 @@ features: {
 
 Events published by the widget. Subscribe to these events using [on](#onevent-callback-context).
 
-- **pageRendered** - triggered when the widget transitions to a new page, and animations have finished.
+### ready
 
-    ```javascript
-    // Overriding the "Back to Sign In" click action on the Forgot Password page
-    signIn.on('pageRendered', function (data) {
-      if (data.page !== 'forgot-password') {
-        return;
-      }
-      var backLink = document.getElementsByClassName('js-back')[0];
-      backLink.addEventListener('click', function (e) {
-        e.preventDefault();
-        e.stopPropagation();
-        // Custom link behavior
-      });
-    });
-    ```
-- **passwordRevealed** - triggered when the show password button is clicked.
+Triggered when the widget is ready to accept user input for the first time. Returns a `context` object containing the following properties:
+
+- **controller** - Current controller name
+
+```javascript
+signIn.on('ready', function (context) {
+  if (context.controller === 'primary-auth') {
+    // The primary authentication form is ready for user input
+  }
+});
+```
+
+### afterRender
+
+Triggered when the widget transitions to a new page and animations have finished. Returns a `context` object containing the following properties:
+
+- **controller** - Current controller name
+
+```javascript
+// Overriding the "Back to Sign In" click action on the Forgot Password page
+signIn.on('afterRender', function (context) {
+  if (context.controller !== 'forgot-password') {
+    return;
+  }
+  var backLink = document.getElementsByClassName('js-back')[0];
+  backLink.addEventListener('click', function (e) {
+    e.preventDefault();
+    e.stopPropagation();
+    // Custom link behavior
+  });
+});
+```
+
+### pageRendered
+
+:warning: This event has been *deprecated*, please use [**afterRender**](#afterrender) instead.
+
+Triggered when the widget transitions to a new page and animations have finished.
+
+```javascript
+signIn.on('pageRendered', function (data) {
+  console.log(data);
+  // { page: 'forgot-password' }
+});
+```
+
+### passwordRevealed
```

Comment:
nit: Wonder if we should deprecate this in favor of an `afterRevealPassword` event.

Reply:
Good call. I'm going to track that with [OKTA-198666](https://oktainc.atlassian.net/browse/OKTA-198666) and will pick that up next.